### DISCUSSION TITLE
Production Use Cases

### DISCUSSION THREAD

**Discussion by SatoriSec** [smallstep/certificates#1157] [General]
Is Step CA used in production use cases or mostly for test or development use cases? Do we have any customer references? Just heard about this tool so just curious.

**Comment by tashian** [comment]
Hi @rajganeshp! Yes, there are many production PKIs running `step-ca` for a wide range of use cases.

We're an 'open core' company, so `step-ca` is also at the heart of [Certificate Manager](https://smallstep.com/certificate-manager/), which extends open source functionality to provide high availability, audit, endpoint reporting and certificate alerting, a management dashboard, and many PKI-related features (active revocation, ACME EAB, granular access control, …) that can scale to the needs of the largest enterprises.

If you're interested in customer references, tap our Intercom widget on https://smallstep.com/ (bottom right corner) and we'd be happy to connect with you on that.
